HOWTO-de.md

```diff
@@ -0,0 +1,17 @@
+Lese das hier auch in anderen Sprachen: [English](HOWTO.md), [Français](HOWTO-fr.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md) [Español](HOWTO-es.md), [فارسی](HOWTO-fa_IR.md)
+
+Willkommen zu *Free-Programming-Books*! Wir heißen neue Beitragende herzlich willkommen; auch die, die ihren ersten Pullrequest (PR) auf Github vor sich haben. Wenn du einer von diesen Personen bist, dann sind hier einige nützliche Ressourcen:
+
+* [Informationen zu Pull Requests](https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+* [Pull Request erstellen](https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+* [Github Hallo Welt](https://guides.github.com/activities/hello-world/) *(auf Englisch)*
+* [Youtube - Tutorial Github für Anfänger](https://www.youtube.com/watch?v=0fKg7e37bQE) *(auf Englisch)*
+* [Youtube - So forkst du ein Github-Repo und sendest einen Pull-Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(auf Englisch)*
+* [Youtube - Github Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(auf Englisch)*
+
+
+Habe keine Angst eine Frage zu stellen; jeder muss irgendwo anfangen und den ersten Pull Request machen. Du könntest der 1000. sein.
+
+Selbst wenn du ein erfahrener Open-Source-Mitwirkender sind, gibt es Dinge, die dich ins Straucheln bringen könnten. Sobald du deinen PR eingereicht hast, führt GitHub Actions einen Linter aus und findet oft kleine Probleme von Absätzen oder Alphabetisierung. Wenn du eine grüne Schaltfläche erhältst, ist alles zur Überprüfung bereit, aber wenn das nicht so ist, klick unter dem fehlgeschlagenen Häkchen auf "Details", um herauszufinden, was dem Linter nicht gefallen hat. Behebe das Problem und füge deinen PR einen Commit hinzu.
+
+Wenn du dir nicht sicher bist, ob die Ressource, die du hinzufügen möchtest, für Free-Programming-Books geeignet ist, lese die Richtlinien in [CONTRIBUTING](CONTRIBUTING.md).
```

Comment:
To be coordinated with @lupinitylabs in their translation (see #5720) ... it's safe change crosslink to `[CONTRIBUTING](CONTRIBUTING-de.md)` too :wink: 

```markdown
Wenn du dir nicht sicher bist, ob die Ressource, die du hinzufügen möchtest, für Free-Programming-Books geeignet ist, lese die Richtlinien in [CONTRIBUTING](CONTRIBUTING-de.md).
```

Reply:
Edited it!